src/runtime_src/core/include/xclbin.h

```diff
@@ -207,7 +211,8 @@ extern "C" {
         uint16_t m_versionPatch;            /* Patch Version */
         uint8_t m_versionMajor;             /* Major Version - Version: 2.1.0*/
         uint8_t m_versionMinor;             /* Minor Version */
-        uint32_t m_mode;                    /* XCLBIN_MODE */
+        uint16_t m_mode;                    /* XCLBIN_MODE */
+        uint16_t m_actionMask;              /* Bit Mask */
```

Comment:
Confirming that this should not break any xclbin on little endian architecture and all our host CPUs are little endian so we should be good.

Reply:
Yes, this piece of code concerned me greatly as well.   So I:
1) Created an xclbin archive image before and after the changes and verify correctness in a binary editor.
2) Additional testing. 
i) Modified the code and set the uint32_t m_mode to 0x1122, validated that the bytes in the xclbin header were: 0x22 0x11, 0x00, 0x00.
ii) Read in modified xclbin and validated that theuint16_t m_mode value was 0x1122.
3) Had multiple code reviews for this piece of code.